public/javascripts/overlays/admin.js

```diff
@@ -327,7 +347,7 @@ function playMatchIntro(flag) {
         'flag': flag,
         'lplayer': document.getElementById('lplayer').value,
         'rplayer': document.getElementById('rplayer').value,
-        'title': document.getElementById('title').value,
+        'tourneyInfo': document.getElementById('tourneyInfo').value,
```

Comment:
In the Admin.hbs file, we should change the display labels (not variable names) of
title -> Tournament Name
tourneyInfo -> Bracket Info

This should prevent any confusion just in case.